runtests.sh

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/bash -ex
+
+gc() {
+  retval=$?
+  docker-compose -f docker-compose.yml down -v || :
+  exit $retval
+}
+trap gc EXIT SIGINT
+
+# Enter local-setup/ directory
+# Run local instances for: dynamodb, gremlin-websocket, gremlin-http
+function start_backbone_service {
+    #pushd local-setup/
+    echo "Invoke Docker Compose services"
+    docker-compose -f docker-compose.yml up  --force-recreate -d
```

Comment:
Do we need the backbone service for gemini server?

Reply:
@msrb The Jenkins file still needs to be worked upon. Updating it. Thanks

Reply:
@GeetikaBatra please rename the function `start_backbone_service` as `start_gemini_service`